samples/XamlTestApplication/Views/MainWindow.paml

```diff
@@ -3,8 +3,11 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" 
         Title="Perspex Test Application" Height="350" Width="525" SizeToContent="WidthAndHeight" >
    
-    <Grid RowDefinitions="Auto,*">
-        <TabControl Grid.Row="1">
+    <Grid RowDefinitions="Auto,*,Auto" ColumnDefinitions="*,*">
+        <TabControl Grid.Row="1" Grid.ColumnSpan="2" Padding="5">
+            <TabControl.Transition>
+				<PageSlide Duration="0.25" />
```

Comment:
Yeah, we used spaces, no tabs. I hadn't actually checked the commit, but the `<PageSlide>` line wasn't aligned with the rest which is usually a symptom of tabs.